keylime/tenant.py

```diff
@@ -543,7 +542,7 @@ def do_cvdelete(self):
             deleted = False
             for _ in range(12):
                 response = httpclient_requests.request("GET", "%s"%(self.cloudverifier_ip), self.cloudverifier_port, params=params, context=self.context)
-                if response.status == 404:
+                if response.status in (200, 404):
```

Comment:
Do we want to match on `200`?  Realistically, a successfully deleted node should be returning `404` at this point I think?

Reply:
reverted.